src/go/rpk/pkg/cli/cmd/redpanda/tune.go

```diff
@@ -91,7 +92,7 @@ func NewTuneCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 				return err
 			}
 			tunerParams.CPUMask = cpuMask
-			conf, err := mgr.FindOrGenerate(configFile)
+			cfg, err := p.Load(fs)
 			if err != nil {
 				if !interactive {
 					return err
```

Comment:
Down below on line 101 this refers to generating a config, which we are removing here. I think we should remove this entire block and just die if we cannot load configs.

Reply:
w.r.t. the flag, we need to mark it as deprecated and not do anything about it, but otherwise I agree on removing the interactive part for this.